src/components/edit-team-form.js

```diff
@@ -172,7 +172,7 @@ export default function EditTeamForm({
               <div className='form-control form-control__vertical'>
                 <label htmlFor='organization'>Add to Organization</label>
                 <Field as='select' name='organization'>
-                  <option value=''>No organization</option>
+                  <option value=''>Select organization</option>
```

Comment:
@LanesGood this suggests that selecting an organization is required, which is not the case. The current option value ("No organization") is also unclear. One possible solution is to add a checkbox to the Add to organization" label and display the dropdown only when it is selected.

Reply:
@vgeorge I agree, this flow isn't so clear. I like your suggestion - I'll see if I can implement this.

Reply:
Updated flow: 
https://user-images.githubusercontent.com/12634024/213499166-30d3fa92-2d6e-464b-85a4-455f95b29a9b.mov